lib/dish.rb

```diff
@@ -0,0 +1,7 @@
+class Dish
+  attr_accessor :name, :price, :description
+
+  def initialize name = 'No Name', price = 'No Price', description = 'No Descr.'
```

Comment:
surely we'd never want to initialise a dish with no name, price etc. so these defaults should be taken out.